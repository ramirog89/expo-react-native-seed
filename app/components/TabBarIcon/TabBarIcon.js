import React from "react";
import PropTypes from "prop-types";

import { Ionicons } from "@expo/vector-icons";

import { COLORS } from "../../constants/application";

const styleTab = {
  marginBottom: -3
};

const TabBarIcon = ({ name, focused }) => {
  return (
    <Ionicons
      name={name}
      size={26}
      style={styleTab}
      color={focused ? COLORS.TAB_SELECTED : COLORS.DEFAULT_BACKGROUND}
    />
  );
};

TabBarIcon.propTypes = {
  name: PropTypes.string,
  focused: PropTypes.bool
};

export default TabBarIcon;
