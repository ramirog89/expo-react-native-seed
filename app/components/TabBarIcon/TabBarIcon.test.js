import React from "react";
import renderer from "react-test-renderer";

import TabBarIcon from "./TabBarIcon";

describe("<TabBarIcon />", () => {
  it("should render", () => {
    const rendered = renderer.create(<TabBarIcon />).toJSON();
    expect(rendered).toMatchSnapshot();
  });
});
