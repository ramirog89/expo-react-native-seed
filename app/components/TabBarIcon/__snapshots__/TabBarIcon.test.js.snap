// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TabBarIcon /> should render 1`] = `<Text />`;
